pkg/logs/input/kubernetes/launcher.go

```diff
@@ -218,6 +243,14 @@ func (l *Launcher) getAnnotation(pod *kubelet.Pod, container kubelet.ContainerSt
 	return ""
 }
 
+//getServiceLabel returns the standard service label for container if present
+func (l *Launcher) getServiceLabel(pod *kubelet.Pod, container kubelet.ContainerStatus) string {
+	if serviceLabel, exists := pod.Metadata.Labels[podStandardLabelService]; exists {
```

Comment:
be carefull, `pod.Metadata.Labels` can be `nil` so it should be check before trying to access a element inside the map.

Reply:
done